include/boost/geometry/algorithms/detail/buffer/buffered_piece_collection.hpp

```diff
@@ -791,7 +775,7 @@ struct buffered_piece_collection
     {
         BOOST_GEOMETRY_ASSERT(boost::size(range) != 0u);
 
-        typename Range::const_iterator it = boost::begin(range);
+        auto it = std::cbegin(range);
```

Comment:
To double-check, is `std` intended here? `boost::size` is called above.

Reply:
Reverted. First I changed some `begin` but later decided it's better to limit this to iterators only. This was a left over.
:heavy_check_mark: 

Unless there are objections, a separate PR could change `boost::begin`  to `std::begin` everywhere (+`end`)

Reply:
I did some more checks, we can't change it everywhere. Because we support geometries adapted to Boost.Range, but not `std::begin` etc.
We might (ever) change that, but that might break behavior for users having custom geometries.